createComments.js

```diff
@@ -233,4 +233,9 @@ const main = async () => {
   // await fs.writeFile(`${config.source.repo}/all-comments.json`, JSON.stringify(comments, null, '  '))
 }
 
+process.on('unhandledRejection', error => {
```

Comment:
I didn't explicitly handle all errors coming from a promise, but I think you can do this instead:

```js
main()
  .catch(err => {
    console.log(err)
  })
```